bin/build-leatherman.ps1

```diff
@@ -0,0 +1,54 @@
+### Set variables from command line
+# $arch => Choose 32 or 64-bit build
+# $cores => Set the number of cores to use for parallel builds
+# $leathermanRef => the git repository to build from
+# $leathermanFork => the git ref to build from
+param (
+[int] $arch=64,
+[int] $cores=2,
+[string] $leathermanRef='origin/master',
+[string] $leathermanFork='git://github.com/puppetlabs/leatherman'
+)
+
+$ErrorActionPreference = 'Stop'
+
+$scriptDirectory = (Split-Path -parent $MyInvocation.MyCommand.Definition);
+. $scriptDirectory\build-helpers.ps1
+. $scriptDirectory\windows-env.ps1
+
+Write-Host "arch=$arch, cores=$cores"
+
+Write-Host "Starting leatherman build"
+
+cd $sourceDir
+
+## Download leatherman and setup build directories
+Invoke-External { git clone $leathermanFork leatherman }
+cd leatherman
+Invoke-External { git checkout $leathermanRef }
+Invoke-External { git submodule update --init --recursive }
+mkdir -Force release
+cd release
+
+## Build leatherman
+$cmake_args = @(
```

Comment:
Going with static leatherman here makes sense. I have a PR we can test eventually for LTH-63.